node-manager/src/nodemanager.rs

```diff
@@ -17,10 +22,23 @@ use crate::{
 pub struct NodeManager {
     mnemonic: Mnemonic,
     wallet: MutinyWallet,
+    node_storage: Mutex<NodeStorage>,
     ws_write: Arc<Mutex<SplitSink<WebSocket, Message>>>,
     counter: usize,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct NodeStorage {
+    pub nodes: Vec<NodeIndex>,
```

Comment:
would this make more sense as a `Map` so we can quickly access specific nodes?

Reply:
done